tests/e2e/utils/kubeflow_installation.py

```diff
@@ -0,0 +1,200 @@
+from pickle import FALSE
+from e2e.utils.utils import load_yaml_file, print_banner
+import argparse
+from e2e.utils.utils import (
+    kubectl_wait_pods,
+    apply_kustomize,
+    install_helm,
+)
+import subprocess
+import os
+
+INSTALLATION_PATH_FILE_VANILLA = "./resources/installation_config/vanilla.yaml"
+INSTALLATION_PATH_FILE_COGNITO = "./resources/installation_config/cognito.yaml"
+
+Install_Sequence = [
+    "cert-manager",
+    "kubeflow-roles",
+    "kubeflow-issuer",
+    "istio-1-14",
+    "kubeflow-namespace",
+    "dex",
+    "cluster-local-gateway",
+    "knative-serving",
+    "knative-eventing",
+    "oidc-authservice",
+    "kubeflow-istio-resources",
+    "kserve",
+    "models-web-app",
+    "central-dashboard",
+    "aws-secrets-manager",
+    "kubeflow-pipelines",
+    "admission-webhook",
+    "jupyter-web-app",
+    "notebook-controller",
+    "volumes-web-app",
+    "training-operator",
+    "katib",
+    "tensorboards-web-app",
+    "tensorboard-controller",
+    "profiles-and-kfam",
+    "user-namespace",
+    #"ack-sagemaker-controller",
+    "ingress",
+    "aws-load-balancer-controller",
+    "aws-authservice",
+]
+
+
+def install_kubeflow(
+    installation_option, aws_telemetry_option, deployment_option, cluster_name
+):
+    INSTALLATION_OPTION = installation_option
+    AWS_TELEMETRY_OPTION = aws_telemetry_option
+    DEPLOYMENT_OPTION = deployment_option
+
+    if DEPLOYMENT_OPTION == "vanilla":
+        path_dic = load_yaml_file(INSTALLATION_PATH_FILE_VANILLA)
+    elif DEPLOYMENT_OPTION == "cognito":
+        path_dic = load_yaml_file(INSTALLATION_PATH_FILE_COGNITO)
+
+    print_banner(
+        f"You are installing kubeflow {DEPLOYMENT_OPTION} deployment with {INSTALLATION_OPTION}"
+    )
+
+    for component in Install_Sequence:
+        build_component(
+            INSTALLATION_OPTION,
+            DEPLOYMENT_OPTION,
+            component,
+            path_dic,
+            cluster_name,
+        )
+
+    if AWS_TELEMETRY_OPTION == "enable":
+        build_component(
+            INSTALLATION_OPTION,
+            DEPLOYMENT_OPTION,
+            "aws-telemetry",
+            path_dic,
+            cluster_name,
+        )
+
+
+def build_component(
```

Comment:
going by other methods in this file, shouldnt this be install_component?

Reply:
ok, can change

Reply:
addressed.